apps/oc_chef_wm/src/oc_chef_wm_routes.erl

```diff
@@ -234,6 +243,8 @@ url_for_search_item_fun(Req, Type, OrgName) ->
 
 make_args(Item, {data_bag, Bag}, OrgName) ->
     [OrgName, Bag, data_bag_item_id(Item)];
+make_args(Item, policies, OrgName) ->
+    [OrgName, ej:get({<<"policy_group">>}, ej:get({<<"name">>}, Item))];
 make_args(Item, client, OrgName) ->
     [OrgName, ej:get({<<"clientname">>}, Item)];
 make_args(Item, _, OrgName) ->
```

Comment:
Since I haven't touched this code in a while, does this work with the default_org_mode stuff? Do we even want it to?